test/dlc_tests/sanity/test_ecr_scan.py

```diff
@@ -221,10 +221,17 @@ def helper_function_for_leftover_vulnerabilities_from_enhanced_scanning(
     remaining_vulnerabilities = ecr_image_vulnerability_list - image_scan_allowlist
     LOGGER.info(f"ECR Enhanced Scanning test completed for image: {image}")
 
-    if remove_non_patchable_vulns and remaining_vulnerabilities:
-        non_patchable_vulnerabilities = extract_non_patchable_vulnerabilities(
-            remaining_vulnerabilities, ecr_enhanced_repo_uri
+    if remove_non_patchable_vulns:
+        non_patchable_vulnerabilities = None
```

Comment:
in what cases do we expect this to stay none? can extract_non_patchable_vulnerabilities() return a false boolean value as well?

any way to rewrite this in if/else format so it's more clear? as written this requires a good amount of file-jumping to understand the purpose of the change

Reply:
Change LGTM, thank you!